package/package.json

```diff
@@ -81,6 +81,7 @@
     "stream-chat": "8.31.0"
   },
   "peerDependencies": {
+    "@react-native-community/netinfo": ">=11.3.2",
```

Comment:
```
    "@react-native-community/netinfo": ">=11.3.2",
```

why so specific? 11.3.2? I dont see that new arch was added there.. why not 11.0 here?

Reply:
ah well I see that it has to be 11.3.1 here 

https://github.com/react-native-netinfo/react-native-netinfo/commit/950410b

Reply:
Yeah, that is why >=11.3.2